src/controllers/index.js

```diff
@@ -2,8 +2,15 @@ const express = require('express');
 
 const router = express.Router();
 const home = require('./home');
+
+const contact = require('./contact');
+const about = require('./about');
+const error = require('./error');
 const levels = require('./levels');
 
-router.get('/', home.get);
-router.get('/levels', levels.get);
-module.exports = router;
+module.exports = router.get('/', home.get)
+  .get('/contact', contact.get)
+  .get('/about', about.get)
+  .get('error404', error.get)
```

Comment:
change the line 14 into `.get('/error404', error.client)` in order to make it work